src/resolvers/taxonomyResolvers.ts

```diff
@@ -111,9 +113,18 @@ export const resolvers = {
       return null;
     },
     async meta(node: Node, _: any, context: ContextWithLoaders): Promise<GQLMeta | null> {
-      if (!node.contentUri?.startsWith("urn:article")) return null;
-      const article = await context.loaders.articlesLoader.load(getArticleIdFromUrn(node.contentUri));
-      return article ? articleToMeta(article) : null;
+      if (node.contentUri?.startsWith("urn:article")) {
+        const article = await context.loaders.articlesLoader.load(getArticleIdFromUrn(node.contentUri));
+        return article ? articleToMeta(article) : null;
+      }
+      return null;
+    },
+    async name(node: Node, _: any, context: ContextWithLoaders): Promise<String | null> {
+      if (node.contentUri?.startsWith("urn:article")) {
+        const article = await context.loaders.articlesLoader.load(getArticleIdFromUrn(node.contentUri));
+        return article ? article.title.htmlTitle : null;
+      }
+      return node.name;
```

Comment:
Vil dette gjøre så alle steder vi spør om navn så vil vi fetche en artikkel?
Burde det vært et annet navn på det som gjør at vi blir litt mer bevisst på det når man skriver en query?

Reply:
Er som sagt usikker på om det er lurt. Målet var å få til språkmerking i strukturdata uten å utvide taksonomi med html-versjoner av navn. Vi kan nok seie at det får vi ikkje til uten masse styr.
Det som kan tale for er at vi allerede henter artikkel via loader for å få ut availability og meta, så om disse hentes sammen med navn så er det ikkje overhead for å få ut navnet.

Reply:
Ah. Du meinte å legge til en ny prop med eget navn slik at vi er bevisst på kva vi henter?

Reply:
Jeg tenkte bare at det er lett å bare spørre om navn uten å få med seg at den faktisk gjør en request.
Men om vi henter ut denne dataen allerede så skal ikke _jeg_ krangle så veldig altså :smile: